api/services/locationService.ts

```diff
@@ -0,0 +1,66 @@
+import { makeSCOSRequest } from './scosService';
+import getLogger from '../utils/logger';
+import { AGENCIES_TABLE } from '../utils/constants';
+import LocationDto from '../models/dto/locationDto';
+import ScosAgencyDto from '../models/scosApi/scosAgencyDto';
+import { notEmpty } from '../utils/typeGuards';
+
+const log = getLogger('locationService');
+
+function mapToLocationDto(agency: ScosAgencyDto): LocationDto | null {
+  // "Site" is an array of 1 element
+  const site = agency.site_info.site[0];
+
+  if (!site) {
+    // there's no location data
+    return null;
+  }
+
+  // as far as I can tell, "Address" has 2 elements but the second is a throwaway
+  const address = site.address[0];
+  const address1 = address.line1;
+  const address2 = `${address.line2} ${address.line3}`.trim();
+
+  // not sure the difference between "st" vs "stg" phones
+  // so we'll just pick the first one that exists
+  const phoneList = [...site.stgphones, ...site.stphones];
+  const phoneNumber = phoneList[0]?.phone ?? '';
+
+  const hours =
+    agency.site_info.detailtext.find(x => x.label === 'Hours')?.text ?? '';
+
+  // TODO: this doesn't exist on this dataset
+  const handicapAccessFlag = 'Y';
+
+  return {
+    id: agency.site_id,
+    address1,
+    address2,
+    zipCode: address.zip,
+    name: site.name,
+    areaCode: '',
```

Comment:
Do we need to return `areaCode` and `phoneExtension` if we don't use them?

Reply:
Currently, the UI breaks if these fields aren't provided.